README.md

```diff
@@ -84,10 +86,15 @@ Supported Cars
 | Honda     | Pilot 2019                    | All               | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Honda     | Ridgeline 2017-19             | Honda Sensing     | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Hyundai   | Elantra 2017-19<sup>1</sup>   | SCC + LKAS        | Stock            | 19mph              | 34mph        |
+| Hyundai   | Elantra GT 2017-19<sup>1</sup>| All               | Stock            | 0mph               | 30mph        |
 | Hyundai   | Genesis 2018<sup>1</sup>      | All               | Stock            | 19mph              | 34mph        |
+| Hyundai   | Ioniq 2017<sup>1</sup>        | All               | Stock            | 0mph               | 34mph        |
+| Hyundai   | Kona 2017-19<sup>1</sup>      | LDWS              | Stock            | 22mph              | 0mph         |
 | Hyundai   | Santa Fe 2019<sup>1</sup>     | All               | Stock            | 0mph               | 0mph         |
 | Jeep      | Grand Cherokee 2016-18        | Adaptive Cruise   | Stock            | 0mph               | 9mph         |
 | Jeep      | Grand Cherokee 2019           | Adaptive Cruise   | Stock            | 0mph               | 39mph        |
+| Kia       | Forte 2018<sup>1</sup>        | LKAS              | Stock            | 0mph               | 0mph         |
```

Comment:
Is SCC default on this?

Reply:
The 2020 base model can have SCC, but it's a package. I can't imagine it was default on a 2018 if it isn't on a 2020. I think it needs to be updated to `SCC + LKAS`.

Reply:
I have a 2019 kia forte that I bought used, it was was originally a rental car, and it does not have SCC. So I would say it is not 'default' unless there is a particular trim for rental cars that i am unaware of

Reply:
Another thing to note with this is that the 2018 kia forte and the 2019 kia forte have completely different specs/tuning (ie steering ratio, mass, etc), but the fingerprints are identical